app/views/home/index.html.erb

```diff
@@ -7,18 +7,13 @@
 
 		<!--Welcome section-->
         <section class="waypoint video-block fullscreen" data-hash="welcome">
-			<%= video_tag(
-				"home/woman-small.mov",
-				id: "womanvideo",
-				autoplay: "autoplay",
-				muted: "muted",
-				loop: "loop",
-				poster: "home/woman_front.jpg",
-				class: "background-video",
-				"x-webkit-airplay": true,
-				"webkit-playsinline": true,
-				preload: "auto"
-			) %>
+
+			<video id="womanvideo" autoplay="autoplay" muted="muted" loop="loop" poster    ="assets/home/woman_front.jpg" 
+			 	    class="background-video" x-webkit-airplay="true" webkit-playsinline="true" preload="auto">
+					<source data-src="assets/home/woman-small.mov" type="video/mp4">
+					<source data-src="assets/home/woman-small.ogm" type="video/ogm">
+			</video>
+
```

Comment:
I don't think it's professional to say things like "你是在搞笑么", "很不专业了" in the code review, because these are subjective and easy to raise the temperature :).  
If there's technical concerns, please bring it up, I'm happy to discuss and collaborate, and re-work on the item.  I know that there're a lot of things I need to catch up and learn from.  
The thing I know is that video_tag is part of the ROR asset pipeline, and cloudfront benefit from it.